node-tests/acceptance/exam-test.js

```diff
@@ -13,7 +13,7 @@ function getNumberOfTests(str) {
   return match && parseInt(match[1], 10);
 }
 
-const TOTAL_NUM_TESTS = 56; // Total Number of tests without the global "Ember.onerror validation tests"
+const TOTAL_NUM_TESTS = 33; // Total Number of tests without the global "Ember.onerror validation tests"
```

Comment:
Its not immediately clear why this number would have changed, whats the reason for this change?

Reply:
Referring to Stef's review on test-load-balance feature pr It seems like upgrading dependencies dropped linting in tests.
https://github.com/ember-cli/ember-exam/pull/136/commits/dde4a68301fa1f3e0c97f4de121bbafb779063a2#r239631433